src/components/ModalDialog/ModalDialog.test.tsx

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022-2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { withAppContext } from 'test/utils'
+
+import ModalDialog from './ModalDialog'
+
+const title = 'foo'
+const content = 'bar'
+const open = true
```

Comment:
does not need to be declaired, just include as a prop how its done right now

Reply:
removed it